nestkernel/spike_data.h

```diff
@@ -63,6 +63,8 @@ class SpikeData
   SpikeData( const SpikeData& rhs );
   SpikeData( const thread tid, const synindex syn_id, const index lcid, const unsigned int lag );
 
+  SpikeData& operator=( const SpikeData& rhs );
```

Comment:
This was to silence a compiler warning through GitHub Actions. If I remember correctly, the problem was that since C++11 if a custom copy constructor or destructor is defined, a custom operator= also needs to be provided or it should be explicitly defined as default.